### PR TITLE
Timeout writes - defaulting to 10s timeout

### PR DESCRIPTION
**Pull request by Prendo93**

### CODE REVIEW

**Review by BlureX — APPROVED** [review]
Should we consider adding a CHANGELOG to this repo? Otherwise LGTM 👍

File: le_test.go [hunk 1]
```diff
@@ -225,26 +225,59 @@ func TestCanSendMoreThan64k(t *testing.T) {
...
 }
 
+func TestTimeoutWrites(t *testing.T) {
+	le, err := Connect("data.logentries.com:443", "myToken")
+	if err != nil {
+		t.Fatal(err)
+	}
+
+	le._testWaitForWrite = &sync.WaitGroup{}
+	le._testWaitForWrite.Add(1) // 3 because we need to write 3 times since it exceeds the limit (64k)
```
Comment (alcamerone) [inline le_test.go]: nit: Copypasta

**Review by alcamerone — APPROVED** [review]
LGTM otherwise

**Comment by Prendo93** [comment]
bump @NicoleHippolite

**Review by NicoleHippolite — APPROVED** [review]

**Pull request merged**